curriculum/challenges/english/25-front-end-development/lab-event-flyer-page/66e45c8140f9fda5c105ae26.md

```diff
@@ -115,10 +117,10 @@ Your `body` element should set its width using `vw`.
 assert(new __helpers.CSSHelp(document).getStyle('body')?.width?.endsWith('vw'));
 ```
 
-Your `body` should use `calc` to set its height to `100vh - 100px`.
+Your `body` should use `calc` to set its min-height to `100vh - 100px`.
```

Comment:
```suggestion
Your `body` should use `calc` to set its `min-height` to `100vh - 100px`.
```
Hmm, I think this should be in backticks...

Reply:
done